sample/hmac.yml

```diff
@@ -3,11 +3,12 @@ provider:
   gateway: http://127.0.0.1:8080
 
 functions:
-  hmac:
-    lang: golang
+  github-hmac:
+    lang: go
     fprocess: handler
-    handler: ./hmac
-    image: alexellis2/hmac
+    handler: ./github-hmac
+    image: alexellis2/github-hmac
     environment:
       secret_key: "keep-this-secret"
+      Http_X-Hub-Signature: "sha1=11"
```

Comment:
This should not be an environment variable in stack.yml, the idea is that it is serialised from the invocation and HTTP header.